### PR TITLE
Add secrets description and fix README

### PR DESCRIPTION
**Pull request by gaiksaya**
Signed-off-by: Sayali Gaikawad <gaiksaya@amazon.com>

### Description
Add secrets description so that it is easy to update them via console. Also fixes a missed step in README due to which jenkins with SSL feature was failing.
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: lib/ci-config-stack.ts [hunk 1]
```diff
@@ -28,12 +28,24 @@ export class CIConfigStack extends Stack {
...
-      const certContentsSecret = new Secret(this, 'certContents', {});
-      const certChainSecret = new Secret(this, 'certChain', {});
-      const privateKeySecret = new Secret(this, 'privateKey', {});
-      const redirectUrlSecret = new Secret(this, 'redirectUrl', {});
-      const OIDCConfigValuesSecret = new Secret(this, 'OIDCConfigValues', {});
+      const arnSecret = new Secret(this, 'certificateArn', {
+        description: 'Certificate ARN retrieved after uploading certificate to IAM server',
+      });
+      const certContentsSecret = new Secret(this, 'certContents', {
+        description: 'Contents of SSL certificate',
```
Comment (peterzhuamazon) [inline lib/ci-config-stack.ts:35]: Suggest: Public Key

Comment (gaiksaya) [reply lib/ci-config-stack.ts]: Fixed. Thanks for the suggestion.

**Review by peterzhuamazon — APPROVED** [review]

File: lib/ci-config-stack.ts [hunk 2]
```diff
@@ -28,12 +28,24 @@ export class CIConfigStack extends Stack {
...
+        description: 'Contents of public key of the SSL certificate',
+      });
+      const certChainSecret = new Secret(this, 'certChain', {
+        description: 'Contents of the SSL certificate chain',
+      });
+      const privateKeySecret = new Secret(this, 'privateKey', {
+        description: 'Contents of private key of the SSL certificate',
+      });
+      const redirectUrlSecret = new Secret(this, 'redirectUrl', {
+        description: 'Redirect url for Jenkins',
```
Comment (abhinavGupta16) [inline lib/ci-config-stack.ts:44]: I think it's a good idea to add a default value as well for this secret as we discussed offline.

**Review by abhinavGupta16 — APPROVED** [review]

File: lib/ci-config-stack.ts (hunk 2 above)
Comment (gaiksaya) [reply lib/ci-config-stack.ts:44]: I did not find a direct way of doing that. I see this feature https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-secretsmanager.SecretStringValueBeta1.html which was implemented `v1.138.0` and later

**Pull request merged**